### ISSUE TITLE
Restructure the editor layout

### ISSUE THREAD

**Issue by KingSupernova31** [KingSupernova31/RulesGuru#185]
It's annoying that you have to constantly scroll up and down on the editor. There's lots of wasted space at the top, so we should collapse the central nevigation column and move the relevant fields upwards.

**Comment by Pulkitxm** [comment]
Can I get assigned? let me know how to start.

**Comment by KingSupernova31** [comment]
Absolutely! The readme has instructions on setting up a local copy of RulesGuru. Let me know if you have any questions. :)
